app/views/administration/orders/_table_row.html.erb

```diff
@@ -31,7 +32,12 @@
           <td><%= order.items.sum("price")%> € </td>
           <td><%= link_to "DOWNLOAD PDF", administration_order_postal_sticker_path(order.id, format: :pdf) %></td>
           <td><%= link_to "DOWNLOAD PDF", home_order_invoice_path(order.id, format: :pdf) %></td>
-          <td><%= order.status%></td>
+          <td>
+            <%= if order.status == 'initiated' then 'Initiée' end %>
+            <%= if order.status == 'shipped' then 'Expédiée' end %>
+            <%= if order.status == 'delivered' then 'Livrée' end %>
```

Comment:
Admin can only ship the package . The status delivered is retro-sent from customer or postal service to confirme the delivery